src/annoylib.h

```diff
@@ -473,28 +475,78 @@ template<typename S, typename T, typename Distance, typename Random>
       _n_items = item + 1;
   }
 
-  void build(int q) {
+  void build_tree_threaded(int num_trees, vector<S> &_roots) {
+	  // just for this thread
```

Comment:
should use two spaces here, not tabs

Reply:
also this function shares a lot of code with `build`, would be nice to merge back